charms/worker/charmcraft.yaml

```diff
@@ -59,15 +55,17 @@ parts:
   charm:
     build-packages: [git]
     charm-entrypoint: k8s/src/charm.py
-  lib:
-    # move the ./k8s/lib path to ./lib since 
-    # charmcraft assumes it to be there once the charm runs
+  promote:
+    # move paths out of ./k8s to ./ since 
+    # charmcraft assumes ./lib to be there
+    # charmcraft assumes ./templates to be there
     after: [charm]
     plugin: nil
     source: ./
     override-prime: |
-      rm -rf $CRAFT_PRIME/lib
+      rm -rf $CRAFT_PRIME/lib $CRAFT_PRIME/templates
       mv $CRAFT_PRIME/k8s/lib $CRAFT_PRIME/lib
+      mv $CRAFT_PRIME/k8s/templates $CRAFT_PRIME/templates
```

Comment:
This moves the templates folder from `./k8s/templates` to `./templates` once installed on the `k8s-worker` units